jaws-journey-deploy/orb.yml

```diff
@@ -327,6 +327,7 @@ commands:
                 account_id: $AWS_ACCOUNT_ID
                 dockerfile: Dockerfile
                 path: ./<< parameters.serviceName >>
+                build-path: ./<< parameters.serviceName >>
```

Comment:
think its build_path

Reply:
https://github.com/CircleCI-Public/aws-ecr-orb/blob/8cf0fb50ab67b51e31df18a1ca9d7aa7dc9aa389/src/jobs/build_and_push_image.yml#L124

Reply:
oh yeah that commit was out of date...